cranelift-codegen/src/ir/dfg.rs

```diff
@@ -117,6 +126,13 @@ impl DataFlowGraph {
     pub fn num_values(&self) -> usize {
         self.values.len()
     }
+
+    /// Starts collection of debug information.
+    pub fn collect_debug_info(&mut self) {
+        if self.values_labels.is_none() {
+            self.values_labels = Some(HashMap::new());
```

Comment:
It took me a while to figure out that I had to do this to track value labels. Could it be documented near `FunctionBuilder::set_val_label` for example?

Reply:
Do you mind opening an issue and re-asking for this there, please? (or add the documentation line you'd expect to see, since this is probably a one-line change)

Reply:
Done